binderhub/static/js/src/repo.js

```diff
@@ -1,18 +1,5 @@
 import { BASE_URL } from "./constants";
 
-let configDict = {};
-
-export function loadConfig(callback) {
-  const req = new XMLHttpRequest();
-  req.onreadystatechange = function() {
-    if (req.readyState == 4 && req.status == 200)
-      callback(req.responseText)
-  };
-  req.open('GET', BASE_URL + "_config", true);
-  req.send(null);
-}
-
-
 function setLabels() {
   const provider = $("#provider_prefix").val();
   const text = configDict[provider]["text"];
```

Comment:
This commit removed `configDict = {}`, but that is seen referenced via `setLabels`. I think something is going to go wrong

Reply:
Ah yes, fixed. I thought I had gotten rid of it, but forgotten I hadn't actually gotten rid of all the places it's used :) coming in a different PR!

Reply:
@consideRatio also I'm not sure if you caught it or the linter did, but eslint would've caught this one as well. So yay for more linting and testing.